src/app/components/LiteSiteCta/index.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '../../contexts/RequestContext';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import styles from './index.styles';
+import { defaultTranslations } from './liteSiteConfig';
+
+type CtaLinkProps = {
+  isRtl: boolean;
+  href: string;
+  text: string;
+  fontVariant?: string;
+  showChevron?: boolean;
+  className?: string;
+};
+
+const CtaLink = ({
+  isRtl,
+  href,
+  text,
+  fontVariant = 'sansRegular',
+  showChevron = false,
+  className,
+}: CtaLinkProps) => {
+  const chevron = isRtl ? (
+    <LeftChevron css={styles.chevron} />
+  ) : (
+    <RightChevron css={styles.chevron} />
+  );
+
+  return (
+    <a href={href} className={className} css={styles.link}>
+      <Text size="brevier" fontVariant={fontVariant} css={styles.linkText}>
+        {text}
+      </Text>
+      {showChevron && chevron}
+    </a>
+  );
+};
+
+const LiteSiteCta = () => {
+  const { dir, translations } = useContext(ServiceContext);
+  const { canonicalLink } = useContext(RequestContext);
+  const isRtl = dir === 'rtl';
+  const { liteSite = defaultTranslations } = translations;
+  const { onboardingMessage, toMainSite, informationPage, dataSaving } =
+    liteSite;
+  const id = 'LiteSiteCta';
+
+  return (
+    <section
+      role="region"
+      data-e2e="lite-cta"
+      aria-labelledby={id}
+      css={styles.outerContainer}
+    >
+      <VisuallyHiddenText as="strong" id={id} aria-hidden>
+        {dataSaving}
+      </VisuallyHiddenText>
+      <div css={styles.container}>
+        <Paragraph>
+          <Text size="brevier">{onboardingMessage}</Text>
+        </Paragraph>
+        <Paragraph>
+          <CtaLink
+            fontVariant="sansBold"
+            isRtl={isRtl}
+            href={canonicalLink}
+            text={toMainSite}
+            css={styles.topLinkSpacing}
+            showChevron
+          />
+        </Paragraph>
+        <Paragraph>
+          <CtaLink
+            isRtl={isRtl}
+            href={canonicalLink}
```

Comment:
This will link to the editorial created page for the Lite site information yea?

Reply:
Ah yep! Will add that as a TODO

Reply:
One question, are we going to link to the lite or canonical article? I guess it's dependant on what editorial put on the page, if there's images for example, we would want to link it as canonical?

Reply:
We will check with product, though I think it makes sense that it links to the .lite site version.

Reply:
Checked and all good for it to go to .lite